CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- `k8sclusterreceiver`: move from github.com/pkg/errors to 'errors' and 'fmt' (#11114)
+- `mongodbatlasreceiver`: move from github.com/pkg/errors to 'errors' and 'fmt' (#11114)
+
```

Comment:
It's not a user facing change, so no need to mention in the changelog. Can you remove this block? I'll add the "Skip changelog" label to pass CI.

Reply:
Hi @dmitryax ,
I fixed it.